# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: C:\pdf995\Lucene-trunk\modules\benchmark\src\java\org\apache\lucene\benchmark\quality
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: QualityStats.java
--- QualityStats.java Base (BASE)
+++ QualityStats.java Locally Modified (Based On LOCAL)
@@ -16,6 +16,7 @@
  */
 package org.apache.lucene.benchmark.quality;
 
+import java.text.DecimalFormatSymbols;
 import java.io.PrintWriter;
 import java.text.NumberFormat;
 import java.util.ArrayList;
@@ -178,7 +179,10 @@
     return (s+padd).substring(0,n);
   }
   private String fracFormat(String frac) {
-    int k = frac.indexOf('.');
+    int k = frac.indexOf(new DecimalFormatSymbols().getDecimalSeparator());
+    // replaced '.' for DecimalFormatSymbols().getDecimalSeparator() in case
+    // default locale uses a different decimal seperator
+    // sorry have been unable to create a testfile for it, but did test in my own environment
     String s1 = padd+frac.substring(0,k);
     int n = Math.max(k,6);
     s1 = s1.substring(s1.length()-n);
